microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -234,23 +325,27 @@ This experiment is available starting with Microsoft Edge version 89.
 *   Progressive Web Apps (PWAs)
 *   Other common web development issues
 
-The [webhint](https://webhint.io) experiment displays the webhint feedback in the [Issues](../issues/index.md) panel.  Select an issue, to display documentation about the solution and a list of the affected resources on your website.  Select a resource link to open the relevant **Network**, **Sources**, or **Elements** pane in DevTools.
+The [webhint](https://webhint.io) experiment displays the webhint feedback in the [Issues](../issues/index.md) panel.  Select an issue to display documentation about the solution and a list of the affected resources on your website.  Select a resource link to open the relevant **Network**, **Sources**, or **Elements** pane in DevTools.
 
 :::image type="content" source="../media/experiments-webhint.msft.png" alt-text="webhint feedback in the Issues panel" lightbox="../media/experiments-webhint.msft.png":::
 
 This experiment is available starting with Microsoft Edge version 85 and is turned on by default.
 
 
 <!-- ====================================================================== -->
-<!-- ## Show issues in Elements -->
+
+## Show issues in Elements
 <!-- present in 96, 98 -->
 
+Needs content.
```

Comment:
This was documented in What's New for 91 here: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/04/devtools#wavy-underlines-highlight-code-issues-and-improvements-in-elements-tool
So we could simply link to it.

Reply:
added short description and link